src/SourceLocator/SourceStubber/ReflectionSourceStubber.php

```diff
@@ -475,7 +476,7 @@ private function parameterDefaultValue(
      */
     private function formatType(CoreReflectionType $type) : NodeAbstract
     {
-        $name     = (string) $type;
+        $name     = $type instanceof CoreReflectionNamedType ? $type->getName() : (string) $type;
```

Comment:
Is the second branch of this conditional ever reached? Mutation tests should indicate if that isn't happening

Reply:
I don't believe so, I'll take a look though and see if I can confirm and remove it

Reply:
That's a little way off yet, maybe just create an issue for now?

Reply:
Yeah, I was mostly looking at what `ReflectionType` instances may pop up. An assertion would be best here, for now.